tests/integration/test_notebooks_gpu.py

```diff
@@ -136,11 +136,11 @@ def test_fastai_integration(notebooks, size, epochs, expected_values):
     [
         (
             15,
-            30,
+            10,
             {
                 "res_syn": {
                     "auc": 0.9716,
```

Comment:
did the auc stay the same?

also I would expect the criteo results to change as well?

Reply:
the results for criteo shouldn't change that much, these are the results in the notebook:
```

at epoch 10 train info: auc:0.7547, logloss:0.4877 eval info: auc:0.747, logloss:0.491
at epoch 10 , train time: 24.0 eval time: 22.5
at epoch 11 train info: auc:0.7559, logloss:0.4869 eval info: auc:0.7476, logloss:0.4904
at epoch 11 , train time: 24.1 eval time: 22.6
at epoch 12 train info: auc:0.7569, logloss:0.4861 eval info: auc:0.7482, logloss:0.49
at epoch 12 , train time: 24.1 eval time: 22.4
at epoch 13 train info: auc:0.7579, logloss:0.4855 eval info: auc:0.7487, logloss:0.4897
at epoch 13 , train time: 24.2 eval time: 22.3
at epoch 14 train info: auc:0.7589, logloss:0.4848 eval info: auc:0.7493, logloss:0.4893
at epoch 14 , train time: 24.3 eval time: 22.5
at epoch 15 train info: auc:0.7601, logloss:0.4841 eval info: auc:0.7498, logloss:0.4889
at epoch 15 , train time: 24.2 eval time: 22.3
at epoch 16 train info: auc:0.7614, logloss:0.4832 eval info: auc:0.7505, logloss:0.4884
at epoch 16 , train time: 24.2 eval time: 22.5
at epoch 17 train info: auc:0.763, logloss:0.4822 eval info: auc:0.7512, logloss:0.4879
at epoch 17 , train time: 24.3 eval time: 22.6
at epoch 18 train info: auc:0.7649, logloss:0.481 eval info: auc:0.752, logloss:0.4874
at epoch 18 , train time: 24.0 eval time: 22.4
at epoch 19 train info: auc:0.767, logloss:0.4795 eval info: auc:0.7528, logloss:0.4867
at epoch 19 , train time: 24.2 eval time: 22.5
at epoch 20 train info: auc:0.7695, logloss:0.4779 eval info: auc:0.7537, logloss:0.4861
at epoch 20 , train time: 24.2 eval time: 22.4
at epoch 21 train info: auc:0.7722, logloss:0.4761 eval info: auc:0.7546, logloss:0.4855
at epoch 21 , train time: 24.1 eval time: 22.3
at epoch 22 train info: auc:0.7751, logloss:0.4742 eval info: auc:0.7554, logloss:0.4849
at epoch 22 , train time: 24.1 eval time: 22.4

```
I'm not that sure about the synthetic data though, in the notebook we had:
```
at epoch 14 train info: auc:0.9967, logloss:0.0859 eval info: auc:0.9676, logloss:0.2369
at epoch 14 , train time: 2.7 eval time: 3.2
at epoch 15 train info: auc:0.9986, logloss:0.0605 eval info: auc:0.9745, logloss:0.2128
at epoch 15 , train time: 2.7 eval time: 3.1

```
but when I computed the integration test I got log_loss=0.6